zarrdataset/_zarrdataset.py

```diff
@@ -6,6 +6,7 @@
 import random
 import zarr
 import numpy as np
+import torch.distributed as dist
```

Comment:
I recommend to import `torch.distributed` after line 42, so the package can still be used even if pytorch is not installed.